### PR TITLE
Issue 5230

### PR DESCRIPTION
**Pull request by be-a-bee**
Add trace_id and span_id to decision logs

there were few conflicts in PR 462 ( https://github.com/open-policy-agent/opa-envoy-plugin/pull/462 ). So creating this to avoid confusion.

### CODE REVIEW

File: internal/internal_test.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ const exampleAllowedRequest = `{
 			"x-b3-sampled": "1",
 			"x-b3-spanid": "537f473f27475073",
 			"x-b3-traceid": "537f473f27475073",
+			"traceparent" : "00-0000000000000000f7bea0490c3cb268-b9543a567b0289f8-01",
+			"tracestate" : "rojo=00f067aa0ba902b7",
```
Comment (ashutosh-narkar) [inline internal/internal_test.go:53]: Where is this used?

**Comment by ashutosh-narkar** [comment]
@be-a-bee please squash your commits and we can then get this in. Thanks!

File: internal/internal_test.go (hunk 1 above)
Comment (be-a-bee) [reply internal/internal_test.go:53]: I intended to use it in a unit test but didn't need it. Can we keep this because it is innocuous ?

**Comment by be-a-bee** [comment]
@ashutosh-narkar  , I squashed the commits and had to create a new PR 
https://github.com/open-policy-agent/opa-envoy-plugin/pull/473 . 

thank you for your patience.

**Comment by ashutosh-narkar** [comment]
Closing in favor of https://github.com/open-policy-agent/opa-envoy-plugin/pull/473.

**Pull request closed without merging**